.github/actions/veracode/action.yml

```diff
@@ -4,20 +4,27 @@ inputs:
   srcclr-api-token:
     description: "Agent API Token"
     required: true
-  srcclr-install-default-options:
-    description: Default options for Source Clear scan
-    required: false
-    default: '-DskipTests -Dmaven.javadoc.skip=true'
-  srcclr-install-options:
-    description: Additional options for Source Clear scan
+  srcclr-project-ext:
+    description: "Veracode project name extension"
```

Comment:
I don't think from here it'll be clear what this project extension is for, IMO we need more documentation

Reply:
I've tried to be more descriptive and lined out sample of the usage in readme file